README.md

```diff
@@ -142,11 +142,10 @@ However, `runPythonTests` does *not* generate the actual reports. For this you n
 
 This function can be used to run the following Python tools for testing:
 
-* bandit
-* black
-* flake8
-* isort
-* mypy
+* Bandit
+* Black
```

Comment:
Isn't `black` part of `ruff`? The acknowledgements section of the documentation, states the following:

"Ruff's autoformatter is built on a fork of Rome's [rome_formatter](https://github.com/rome/tools/tree/main/crates/rome_formatter), and again draws on both the APIs and implementation details of [Rome](https://github.com/rome/tools), [Prettier](https://github.com/prettier/prettier), and [Black](https://github.com/psf/black)."